### DISCUSSION TITLE
SDG15 research

### DISCUSSION THREAD

**Discussion by salty-tree** [salty-tree/dgj-2024-dcspd#3] [General]
### Research topics
Main theme: SDG 15 - **Life on Land**

Target 15.3
END DESERTIFICATION AND RESTORE DEGRADED LAND
By 2030, combat desertification, restore degraded land and soil, including land affected by desertification, drought and floods, and strive to achieve a land degradation-neutral world.

Target 15.4
ENSURE CONSERVATION OF MOUNTAIN ECOSYSTEMS
By 2030, ensure the conservation of mountain ecosystems, including their biodiversity, in order to enhance their capacity to provide benefits that are essential for sustainable development.

THINGS TO DO

- Don’t shop for pets! Visit your local animal shelter and adopt an animal there.

(I have an idea here that after you complete a hub u get a pet/follower to perhaps symbolize that u adopted an animal or smth)

- Volunteer with street animal organizations or shelters: Help to sustain diversity and save lives of animal species by volunteering in organizations helping these pets as well as endangered species.

_Originally posted by @VectorScore in https://github.com/n0Oo0Oo0b/dgj-2024-dcspd/discussions/1#discussioncomment-8334539_

**Comment by VectorScore** [comment]
https://www.globalgoals.org/take-action/?id=15
